<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="order.createOrderByAddingProductToCart.CreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD" tests="1" time="171.354" ignored="0" failures="0" timestamp="06 Mar 2018 08:18:17 GMT" skipped="0" hostname="NamTH2-PC2" errors="1">
  <testcase name="C005_VerifyWhenCreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD" time="171.354" classname="order.createOrderByAddingProductToCart.CreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at automationLibrary.actions.ActionGenWeb.selectItemFromSelect2(ActionGenWeb.java:581)
	at ShoppingFlow.actions.Action.addNewRecipient_checkoutpage(Action.java:594)
	at ShoppingFlow.actions.Action.checkout(Action.java:682)
	at ShoppingFlow.actions.Action.checkOutOrder(Action.java:1210)
	at order.createOrderByAddingProductToCart.CreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD.C005_VerifyWhenCreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD(CreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- C005_VerifyWhenCreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD -->
</testsuite> <!-- order.createOrderByAddingProductToCart.CreateOrderFromShopByAddingProductToCart_HomePage_NotLogin_COD -->
